gc.md

```diff
@@ -72,17 +85,37 @@ everything else is garbage*
 - objects pointed to by local variables are roots (stack is scanned for roots)
 - DOM elements are roots (may be weakly referenced)
```

Comment:
DOM objects now participate in marking similar to JS objects: https://v8project.blogspot.de/2018/03/tracing-js-dom.html

Reply:
Addressed in be7aef2